lib/controllr.rb

```diff
@@ -1,9 +1,11 @@
-require 'json'
 require 'geocoder'
 require 'redis'
 require './lib/age'
+require './lib/json_api'
 
 class Controllr
+  API_ENDPOINT = 'http://controllr.panter.biz'
```

Comment:
I added an item to our Technik todo list to finally switch Controllr to HTTPS ;-)